app/api/helpers/order.py

```diff
@@ -15,3 +19,19 @@ def delete_related_attendees_for_order(self, order):
         except Exception as e:
             logging.error('DB Exception! %s' % e)
             self.session.rollback()
+
+
+def set_expiry_for_order(order, override=False):
+    """
+    Expire the order after the time slot(10 minutes) if the order is pending.
+    Also expires the order if we want to expire an order regardless of the state and time.
+    :param order: Order to be expired.
+    :param override: flag to force expiry.
+    :return:
+    """
+    if order and not order.paid_via and (override or (order.status == 'pending' and (
+                order.created_at +
+                timedelta(minutes=ticketing.TicketingManager.get_order_expiry())) < datetime.now(timezone.utc))):
```

Comment:
Why not just use `datetime.utcnow()`?

Reply:
For some reason it doesn't work :confused:  It complains about it not being timezone aware. Hence had to use it like this. Hardly any difference though.

Reply:
Yes, what you have used is timezone aware:
```
>>> datetime.now(timezone.utc)
datetime.datetime(2018, 7, 13, 20, 57, 47, 167760, tzinfo=datetime.timezone.utc)
>>> datetime.utcnow()
datetime.datetime(2018, 7, 13, 20, 57, 55, 988286)
```